### ISSUE TITLE
In Mobile-View the search field and navigation are not accessible when tab is pressed

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#599]
In **desktop view**:
- [x] Search close button "X" should be accessible when tab is pressed 

In **mobile view**, the search field is not accessible when tab is pressed. 

- [x] Search field shouldn't be part of the tag "nav", because an impared user don't find and don't expect the search inside the navigation
- [x] When search field gets the focus with tab, it should get open and be ready to search
- [x] Search close button "X" is missing, please add
- [ ] Navigation menu points are not accessible with the tab key (or any key)
 
![image](https://user-images.githubusercontent.com/7510842/32959352-a7c90b30-cbc1-11e7-9527-33cb6fa5e5a1.png)
